nerima/src/main/java/brocoeur/example/nerima/controller/NerimaController.java

```diff
@@ -27,26 +24,16 @@ public class NerimaController {
     private RabbitTemplate rabbitTemplate;
 
     @PostMapping("/api/nerima/gamble")
-    public ResponseEntity<ServiceRequest> postDirectGamblePlay(@RequestBody final ServiceRequest serviceRequest) {
+    public ResponseEntity<ServiceRequest> postSinglePlayerGamblePlay(@RequestBody final ServiceRequest serviceRequest) {
```

Comment:
What about post multiplayer gamble plays ? Should be the same url but where do you make the distinction ?

Reply:
for now it is same url for single and multiplayer games, dividing them is a good idea, will create a ticket.